docker-local-mojaloop/docker-compose.yml

```diff
@@ -0,0 +1,424 @@
+version: "3.7"
```

Comment:
there is a need for the additional entry in the readme.md file where the layout of services specified in docker-compose.yml is visualized. There are so many services so it should be clear for a newbie which local port is connected to which service.

Reply:
yes. I agree. I plan to add an on-boarding doc with visuals and explanations.
lets leave this for another issue. I plan to make user stories to address changes.

Reply:
yes, let do this in separate issue!